frontend/src/app/modules/boards/board/inline-add/board-inline-add-autocompleter.component.ts

```diff
@@ -120,6 +113,12 @@ export class BoardInlineAddAutocompleterComponent implements AfterContentInit {
       filters.add('id', '!', rows.map((wp:WorkPackageResource) => wp.id!));
     }
 
+    // Return when the search string is empty
```

Comment:
@HDinger These lines look good to me, but is there a strong reason, why these lines are not the first ones in this method? We could save useless execution.